spec/api/v1/profiles_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+describe 'Profiles API' do
+  describe 'GET /' do
+    context 'unauthorized' do
+      it 'returns 401' do
+        get '/api/v1/profiles/me', format: :json
+        expect(response.status).to eq 401
+      end
+    end
+
+    context 'authorized' do
+      let!(:me) { create(:user) }
+      let!(:users) { create_list(:user, 4) }
+      let(:access_token) { create(:access_token, resource_owner_id: me.id) }
+
+      before { get '/api/v1/profiles', format: :json, access_token: access_token.token }
+
+      it 'returns 200 status' do
+        expect(response).to be_success
+      end
+
+      it 'list of users without me' do
+        save_and_open_page
+        expect(response.body).to be_json_eql(users.to_json)
```

Comment:
Started GET "/api/v1/profiles/me" for 127.0.0.1 at 2015-11-30 15:16:31 +0200
Processing by Api::V1::ProfilesController#me as HTML
Filter chain halted as :doorkeeper_authorize! rendered or redirected
Completed 401 Unauthorized in 1ms (ActiveRecord: 0.0ms)